system/templates/rolls/ironsworn-roll-message.hbs

```diff
@@ -10,49 +13,60 @@
   <section class='roll-outcome {{outcomeClass}}'>
     <h4 class='outcome-label' title='{{outcome.source}}'>
       {{#if outcomeReplacementReason}}
-      <span class='outcome-replacement-reason'>
-        {{outcomeReplacementReason}}:&nbsp;<wbr />
-      </span>
+        <span class='outcome-replacement-reason'>
+          {{outcomeReplacementReason}}:&nbsp;<wbr />
+        </span>
       {{/if}}
       <span class='outcome-text'>
         {{outcomeText}}
       </span>
     </h4>
 
     {{#if moveOutcome}}
-    <section class='move-outcome'>
-      {{{enrichMarkdown moveOutcome}}}
-    </section>
+      <section class='move-outcome'>
+        {{{enrichMarkdown moveOutcome}}}
+      </section>
```

Comment:
OK, I think I've got my VS Code configured to use prettier for handlebars, which seems to match this whitespace preference. Hopefully we won't keep doing this in the future. :D

Reply:
reckon it'd be do a separate prettier formatting PR to catch all of those in one go?